gnucash/ui/gnc-plugin-page-register.ui

```diff
@@ -294,7 +294,7 @@
     <item>
       <attribute name="label" translatable="yes">Sche_dule…</attribute>
       <attribute name="action">GncPluginPageRegisterActions.ScheduleTransactionAction</attribute>
-      <attribute name="tooltip" translatable="yes">Create a Scheduled Transaction with the current transaction as a template</attribute>
+      <attribute name="tooltip" translatable="yes">Create a Scheduled Transaction with the current transaction as a template or edit the Scheduled Transaction that current transaction was created by</attribute>
```

Comment:
Please finish the sentence with a full stop `.`!

Reply:
Are you sure? I know the grammar, but there are no full stops at the end of strings/sentences in this file (except for the line 99).

Reply:
It would be nice, if you fixed all then.
In the times of DOS almost nobody applied them, but today [weblate [old message]](https://hosted.weblate.org/translate/gnucash/gnucash/en/?checksum=fd06b08929b6ca69&q=Create+a+Scheduled+Transaction+with&sort_by=-priority%2Cposition) is sending us warnings.